src/marta/index.js

```diff
@@ -20,6 +29,38 @@ class Marta {
 
     return list;
   }
+
+  static serviceId() {
+    let serviceId = this.defaultServiceId();
+
+    // check exceptions
+    const today = this.formatDate(new Date());
+    const exceptions = serviceExceptions.filter(exception => exception.date.toString() === today);
+    if (exceptions.length) {
+
+
+    }
+    return serviceId;
+  }
+
+  // private
+  static defaultServiceId() {
+    const today = new Date();
+    const day = today.getDay();
+    switch(day) {
+      case 6: // Saturday
+        return 3;
+      case 0: // Sunday
+        return 4;
+      default: // weekdays
+        return 5;
+    }
```

Comment:
Oh snap, you already know about service_ids and exceptions. I didn't need to type out that big huge comment. :tada: